src/components/Session/Session.tsx

```diff
@@ -11,13 +12,15 @@ import {
 export const useSession = () => {
   const { data, loading: sessionLoading } = useSessionQuery();
   const session = data?.session.user;
+  const powers = data?.session.powers;
 
-  return { session, sessionLoading };
+  return { session, sessionLoading, powers };
```

Comment:
Powers is already in the session object. I think the moving it up here is redundant.

Reply:
The `session` is actually just the user as you can see from above 

`const session = data?.session.user;`

I thought about renaming it but it's used in alot of files, and wasn't sure if it's named as `session` instead of `user` in the first place for a reason?  Let me know what you think

Reply:
Ah I see. It's only used in a few spots anyways so let's just leave it.